chart/values.yaml

```diff
@@ -0,0 +1,295 @@
+      podSecurityContext:
+        enabled: false
+      containerSecurityContext:
+        enabled: false
+    provisioning:
+      podSecurityContext:
+        enabled: false
+      containerSecurityContext:
+        enabled: false
+  # TPA's file storage configuration, points back to Minio, defined previously.
+  storage:
+    endpoint: http://tpa-infrastructure-minio:9000
+    accessKey:
+      valueFrom:
+        secretKeyRef:
+          name: tpa-infrastructure-minio
+          key: root-user
+    secretKey:
+      valueFrom:
+        secretKeyRef:
+          name: tpa-infrastructure-minio
+          key: root-password
+  # TPA's event-bus configuration, points back to Kafka, defined previously.
+  eventBus:
+    type: kafka
+    bootstrapServers: tpa-infrastructure-kafka:9092
+    config:
+      securityProtocol: SASL_PLAINTEXT
+      username: "user1"
+      password:
+        valueFrom:
+          secretKeyRef:
+            name: tpa-infrastructure-kafka-user-passwords
+            key: client-passwords
+      mechanism: PLAIN
+  # TPA's OIDC (OpenID Connect) configuration.
+  oidc:
+    clients:
+      frontend: {}
+      walker:
+        clientSecret:
+          # OIDC client secret for the Walker module.
+          value: |-
+            ${TPA__OIDC__WALKER_CLIENT_SECRET}
+      testingUser:
+        clientSecret:
+          # OIDC client secret for the Testing User module.
+          value: |-
+            ${TPA__OIDC__TESTING_USER_CLIENT_SECRET}
+      testingManager:
+        clientSecret:
+          # OIDC client secret for the Testing Manager module.
+          value: |-
+            ${TPA__OIDC__TESTING_MANAGER_CLIENT_SECRET}
+  # TPA's modules configuration.
+  modules:
+    bombasticWalker:
+      sources:
+        redhat:
+          url: https://access.redhat.com/security/data/sbom/beta/
+          signingKeyUrl: https://access.redhat.com/security/data/97f5eac4.txt#77E79ABE93673533ED09EBE2DCE3823597F5EAC4
+          acceptV3Signatures: true
+          fixLicenses: true
+          job:
+            schedule: "0 * * * *"
+    vexinationWalker:
+      sources:
+        redhat:
+          url: https://www.redhat.com/.well-known/csaf/provider-metadata.json
+          acceptV3Signatures: true
+          ignoreDistributions:
+            - https://access.redhat.com/security/data/csaf/v2/advisories/
+          job:
+            schedule: "0 * * * *"
+  # TPA's GUAC (Graph for Understanding Artifact Composition) configuration.
+  guac:
+    database:
+      name: guac
+      host: tpa-infrastructure-postgresql
+      sslMode: disable
+      username: guac
+      # GUAC administrator (guac) password.
+      password: |-
+        ${TPA__GUAC__PASSWORD}
+    initDatabase:
+      name: postgres
+      host: tpa-infrastructure-postgresql
+      username: postgres
+      sslMode: disable
+      password:
+        valueFrom:
+          secretKeyRef:
+            name: tpa-infrastructure-postgresql
+            key: postgres-password
```

Comment:
We'll need to prune TPA blocks out.

```suggestion
```

Reply:
I have these changes here: https://github.com/Jdubrick/rhdh-installer/commit/c51b7a3c3d47e7ac38e75cab6e877d6919248b8b

I just wanted to get a quick PR up for the structure so I copied the files raw from RHTAP. I can include the changes in this PR if we'd prefer?

Reply:
Yeah I think it'll be good to have these changes in this PR.